config/authorization/config.ex

```diff
@@ -16,6 +16,7 @@ defmodule Acl.UserGroups.Config do
                         "http://data.lblod.info/vocabularies/erediensten/EredienstMandataris",
                         "http://data.lblod.info/vocabularies/erediensten/BestuurVanDeEredienst",
                         "http://data.lblod.info/vocabularies/erediensten/EredienstBestuurseenheid",
+                        "http://data.vlaanderen.be/ns/besluit#Bestuursorgaan",
```

Comment:
it is already on @org_type, this shouldn't be there I think. and it might produces issues (writing on multiple graph).

Reply:
Ah ! Indeed !